packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5133,6 +5133,981 @@ curl -v -X GET \
 }
 ```
 
+## Application client secret management operations
+
+### Add new client secret
+
+<ApiOperation method="post" url="/api/v1/apps/${applicationId}/credentials/secrets" />
+
+Adds a new secret to the client's collection of secrets
+
+> **Note:** This API lets you bring your own secret. If no secrets are specified in the request, it adds a new system-generated secret.
+
+##### Request parameters
+
+| Parameter     | Description                                                                     | Param Type | DataType                                      | Required | Default |
+| ------------- | ------------------------------------------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
+| applicationId | Unique key of an [Application](#application-object)                                 | URL        | String                                    | TRUE     |         |
+| metadata | Metadata for creating a secret [Client Secret Metadata](#client-secret-request-object)   | Body      | Client Secret Metadata                     | FALSE    |         |
```

Comment:
Can we change the description to be easier to understand that this is in fact the data you need to provide to specify the secret? How about `Client secret request object`?

Reply:
updated